tests-configuration-infinispan/src/test/java/org/jboss/hal/testsuite/test/configuration/infinispan/remote/cache/container/configuration/AttributesTest.java

```diff
@@ -67,6 +69,7 @@ public void editDefaultRemoteCluster() throws Exception {
     }
 
     @Test
+    @Ignore
```

Comment:
What is the reason for ignore?

Reply:
UI model change, attribute (that is subject of testing) became deprecated